src/components/datagrid/data_grid.tsx

```diff
@@ -940,11 +940,14 @@ export const EuiDataGrid: FunctionComponent<EuiDataGridProps> = (props) => {
     </Fragment>
   );
 
-  // When data grid is full screen, we add a class to the body to remove the extra scrollbar
-  if (isFullScreen) {
-    document.body.classList.add('euiDataGrid__restrictBody');
-  } else {
-    document.body.classList.remove('euiDataGrid__restrictBody');
+  // need to safely access those Web APIs
+  if (typeof window !== 'undefined') {
```

Comment:
This _should_ always work, especially in correctly configured node environments like jest's default jsdom setup, but it'll be more future proof & environment proof to test for the accessed global:

```suggestion
  if (typeof document !== 'undefined') {
```